### PR TITLE
carbmee user, solution and tests

### PR DESCRIPTION
**Pull request by lelecanfora**

### CODE REVIEW

File: catalog/conformance-tests/result-innoblocktech-001.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "tested_solution": {
+    "solution_id": "innoblocktech",
+    "version": "1.0"
```
Comment (raimundo-henriques) [inline catalog/conformance-tests/result-innoblocktech-001.json:4]: Please note that versions must have three digits (following semver), e.g. (please replace with the actual version):
```suggestion
    "version": "1.0.0"
```

File: catalog/conformance-tests/result-innoblocktech-001.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  },
+  "tested_by": {
+    "solution_id": "carbmee",
+    "version": "1.0"
```
Comment (raimundo-henriques) [inline catalog/conformance-tests/result-innoblocktech-001.json:8]: Same as above (please replace with the actual version): 
```suggestion
    "version": "1.0.0"
```

**Review by raimundo-henriques — CHANGES_REQUESTED** [review]
Thank you for the PR @lelecanfora!

There is, however, an issue regarding your `solution.json`: its contents actually concern ClimateCamp. This was probably a working version that by mistake made it through to the PR?

CC.: @prasadt1

File: catalog/conformance-tests/result-innoblocktech-001.json (hunk 1 above)
Comment (lelecanfora) [reply catalog/conformance-tests/result-innoblocktech-001.json]: both have been fixed in the latest commit

File: catalog/conformance-tests/result-innoblocktech-001.json (hunk 2 above)
Comment (lelecanfora) [reply catalog/conformance-tests/result-innoblocktech-001.json]: both have been fixed in the latest commit

**Pull request merged**